### DISCUSSION TITLE
Max allowed claims based on permission

### DISCUSSION THREAD

**Discussion by isaaclepes** [GriefPrevention/GriefPrevention#1166] [Ideas]
Forgive me if this feature exists, but I could not find it documented.

As a server owner, I would like to award additional claims to players as they rank-up using simple permissions.
This would allow me to reward dedicated players with additional claims, based of permission group.

Suggested format:
griefprevention.claimcount.# where # is the allowed number of claims a player may have.

Example:
Default - No claims allowed - permission griefprevention.createclaims false
Rank 1 - 1 Claim allowed - permission for group griefprevention.claimcount.1
Rank 2 - 2 Claims allowed - permission for group griefprevention.claimcount.2
Rank 3 - 3 Claims Allowed - permission for group griefprevention.claimcount.3
Rank 4 - 4 Claims Allowed - permission for group griefprevention.claimcount.4

Right now, it appears that we can only set a global limit and then give permission to allow unlimited claims using griefprevention.overrideclaimcount

**Comment by RoboMWM** [comment]
ya this is a dupe of something somewhere
